packages/snaps-simulation/src/simulation.ts

```diff
@@ -466,6 +492,10 @@ export function getPermittedHooks(
     getSnapState: getPermittedGetSnapStateMethodImplementation(runSaga),
     updateSnapState: getPermittedUpdateSnapStateMethodImplementation(runSaga),
     clearSnapState: getPermittedClearSnapStateMethodImplementation(runSaga),
+
+    getSnap: getGetSnapImplementation(true),
```

Comment:
Are we intending on preinstalled being true for every Snap using snaps-jest?

Reply:
Yes, since it's only used for the methods restricted to preinstalled Snaps. We can make it configurable but I don't know if it's worth the effort?

Reply:
Fine to default to true for now